Configure.ps1

```diff
@@ -4,8 +4,8 @@
 # Before running this script you need to install the Azure RM cmdlets as an administrator. 
 # For this:
 # 1) Run Powershell as an administrator
-# 2) in the PowerShell window, type: Install-Module AzureRM.Resources
-#
+# 2) in the PowerShell window, type: Install-Module AzureAD
+
 # To prepare this script
 # 3) With the Azure portal (https://portal.azure.com), choose your active directory tenant, then go to the Properties of the tenant and copy
 #    the DirectoryID. This is what we'll use in this script for the tenant ID
```

Comment:
Yes, Blair. My intention was to:
- change the Readme.md to extract the manual process to another .md file (for people who want to understand how to do that manually)
- replace the content of the manual process by explanations on how to run the script, and how to cleanup.